proto/aggkit/prover/v1/aggchain_proof_generation.proto

```diff
@@ -44,3 +44,17 @@ message GenerateAggchainProofResponse {
   // Custom chain data.
   bytes custom_chain_data = 5;
 }
+
+message ImportedBridgeExitWithBlockNum {
+  // The block number of the imported bridge exit.
+  uint64 block_number = 1;
+  // The imported bridge exit.
+  agglayer.protocol.types.v1.ImportedBridgeExit imported_bridge_exit = 2;
+}
+
+message ClaimFromMainnetWithBlockNumber {
+  // The block number of the claim.
+  uint64 block_number = 1;
+  // The claim.
+  agglayer.protocol.types.v1.ClaimFromMainnet claim_from_mainnet = 2;
```

Comment:
Actually this isn't supposed to be the claims but the inserted GER. The claims are the `Imported bridge exit`

so we should take advantage of this api change to rename this `ClaimFromMainnet` to something else to avoid confusion because it is actually not a claim from mainnet but simply data to verify one inserted GER against one L1 info root.

I'd say to rename this `ClaimFromMainnet` to `InsertedGER` to align with the spec/program terminology

Reply:
fixed